Phases/Assessments/README.md

```diff
@@ -85,3 +88,14 @@ To prepare for this assessment, learners should consider the following questions
 - Why did you make this design decision?
 - What are the inputs and outputs?
 - How did you arrive at your solution? Walk through your commit history to show how your solution grew.
+
+### Membership Review
+
+After the evaluation & interview are completed, Membership will make a decision and inform the learner.
+
+In most cases, the decision will be either one of these two options:
+
+- **Pass**: learner continues on to the next phase
+- **Fail**: learner begins membership exit process
+
+If, however, the learner completes and fails the assessment _early_ (i.e. before the 8th week of their phase), then they can re-take the assessment again on or before the 8th week of their phase. Learners can only re-take an assessment once.
```

Comment:
This is more concise, but again, is written in such a way that "failing early" seems like the exception rather than the rule. If the pacing expectation is really 5 weeks of content followed by assessment in the 6th week, "failing early" should be the rule rather than the exception.